docs/development_guide.md

```diff
@@ -88,6 +151,16 @@ For other projects you can use the `--project` and `--gs_dir` options of the
 script.
 
 ### Pushing changes to your fork's branch
+
+Before pushing changes, make sure the pylint checks pass. To install pylint:
+```bash
+.[PATH_TO_VENV]/bin/activate
+pip install pylint
```

Comment:
nit: consider adding `--upgrade` to be consistent with elsewhere.

Reply:
Done.